oqamaint/osclib

```diff
@@ -0,0 +1 @@
+../osclib
```

Comment:
Again, this seems less than ideal. My understanding is that since you are defining the subdir as a package you can use the following.

```python
from ..osclib import stagingapi
```

Reply:
as above, relative imports works only inside package..